file/types.go

```diff
@@ -103,10 +103,10 @@ func copyToService(fService FService) service {
 func unwrapURL(urlString string, fService *FService) error {
 	parsed, err := url.Parse(urlString)
 	if err != nil {
-		return errors.New("invaid url: " + urlString)
+		return fmt.Errorf("invaid url: " + urlString)
```

Comment:
Would it be an opportunity to use verbs here and in the rest of the file ?

Reply:
Same response as the other comment.

Reply:
Ok, but it seems there is a typo here, by the way